### ISSUE TITLE
[Web]

### ISSUE THREAD

**Issue by buimanhhungit** [singerdmx/flutter-quill#199]
Up, down, left, right key not working some case

With space key, I follow your instructions but not working.
https://user-images.githubusercontent.com/51559988/116678217-42de7180-a9d3-11eb-8005-1ca43528d4e8.png

Note: Flutter 2.0

**Comment by GyuriMajercsik** [comment]
I see. What we want to achieve is to remove the faulty shortcuts from the `MaterialApp`. 
To be null-safety compatible you can do the following: 

```
MaterialApp(
      title: 'Deescus',
      shortcuts: ignoredNavigationShortcuts(), 
.... 

Map<LogicalKeySet, Intent> ignoredNavigationShortcuts() {
  Map<LogicalKeySet, Intent> shortcuts = Map.from(WidgetsApp.defaultShortcuts);
  if (kIsWeb) {
    shortcuts.remove(LogicalKeyboardKey.space);
    shortcuts.remove(LogicalKeyboardKey.arrowUp);
    shortcuts.remove(LogicalKeyboardKey.arrowDown);
    shortcuts.remove(LogicalKeyboardKey.arrowLeft);
    shortcuts.remove(LogicalKeyboardKey.arrowRight);
  }
  return shortcuts;
}

```

**Comment by buimanhhungit** [comment]
I checked. it doesn't work properly when wrapping by listview...
How to fix it?

**Comment by GyuriMajercsik** [comment]
Please open concrete issues with code samples so that we can fix it in Quill. 
The problem you encounter in Listview might not be related to Quill. So, please remember, the more information you give us, the easier is to fix the problem. 
Also, pull requests are always welcomed. 

Thanks 😊

**Comment by NicolasDionB** [comment]
If anyone comes here, it has been fixed in version 3.8.2, PR #618 .
